### ISSUE TITLE
[Bug] Can not start CE in Ubuntu 20.04.3 LTS

### ISSUE THREAD

**Issue by gilbertoe** [thingsboard/thingsboard#5264]
Hi

I have followed the instruction provided @ https://github.com/thingsboard/thingsboard/issues/5244#issue-996346963  and now  I managed to finish the installation procedure. However, I can not acesss my ThingsBoard installation @   http://localhost:8080/.

I have started ThingBoard with the command "sudo service thingsboard start" with no errors, but it seems it is not running at all.

gilberto@kerberos:~$ sudo service thingsboard start
gilberto@kerberos:~$ service --status-all
 [ + ]  acpid
 [ - ]  alsa-utils
 [ - ]  anacron
 [ + ]  apparmor
 [ + ]  apport
 [ + ]  avahi-daemon
 [ + ]  binfmt-support
 [ + ]  bluetooth
 [ - ]  bootmisc.sh
 [ - ]  brltty
 [ - ]  checkfs.sh
 [ - ]  checkroot-bootclean.sh
 [ - ]  checkroot.sh
 [ - ]  console-setup.sh
 [ + ]  cron
 [ + ]  cups
 [ + ]  cups-browsed
 [ + ]  dbus
 [ - ]  dns-clean
 [ + ]  gdm3
 [ - ]  grub-common
 [ + ]  hddtemp
 [ - ]  hostname.sh
 [ - ]  hwclock.sh
 [ + ]  irqbalance
 [ + ]  kerneloops
 [ - ]  keyboard-setup.sh
 [ - ]  killprocs
 [ + ]  kmod
 [ - ]  lightdm
 [ + ]  lm-sensors
 [ + ]  mono-xsp4
 [ + ]  mosquitto
 [ - ]  mountall-bootclean.sh
 [ - ]  mountall.sh
 [ - ]  mountdevsubfs.sh
 [ - ]  mountkernfs.sh
 [ - ]  mountnfs-bootclean.sh
 [ - ]  mountnfs.sh
 [ - ]  mysql
 [ + ]  network-manager
 [ + ]  networking
 [ - ]  ondemand
 [ + ]  openvpn
 [ - ]  plymouth
 [ - ]  plymouth-log
 [ + ]  postgresql
 [ - ]  pppd-dns
 [ + ]  procps
 [ - ]  pulseaudio-enable-autospawn
 [ + ]  rc.local
 [ + ]  resolvconf
 [ - ]  rsync
 [ + ]  rsyslog
 [ - ]  saned
 [ - ]  sendsigs
 [ - ]  speech-dispatcher
 [ - ]  spice-vdagent
 [ + ]  sysstat
 [ + ]  thermald
 [ + ]  udev
 [ + ]  ufw
 [ - ]  umountfs
 [ - ]  umountnfs.sh
 [ - ]  umountroot
 [ + ]  unattended-upgrades
 [ - ]  urandom
 [ - ]  uuidd
 [ - ]  warsaw
 [ + ]  whoopsie
 [ - ]  x11-common

Thanks in advance
Gilberto

**Comment by Backdraft007** [comment]
`service --status-all` says nothing

Use this:
`sudo systemctl status thingsboard`

And have a look in the logfile. /var/log/thingsboard/thingsboard.log

**Comment by gilbertoe** [comment]
Hi

Here is the result of the command and the contend of the log file. I did not see any ṕroblems. 

gilberto@kerberos:~$ sudo service thingsboard start
gilberto@kerberos:~$ sudo systemctl status thingsboard
● thingsboard.service - thingsboard
     Loaded: loaded (/lib/systemd/system/thingsboard.service; enabled; vendor preset: enabled)
     Active: active (running) since Wed 2021-09-22 21:13:34 -03; 4s ago
   Main PID: 36878 (thingsboard.jar)
      Tasks: 24 (limit: 9339)
     Memory: 334.6M
     CGroup: /system.slice/thingsboard.service
             ├─36878 /bin/bash /usr/share/thingsboard/bin/thingsboard.jar
             └─36912 /usr/bin/java -Dsun.misc.URLClassPath.disableJarChecking=true -Dplatform=deb -Dinstall.data_dir=/usr/share/thingsboard/d>

set 22 21:13:34 kerberos systemd[1]: Started thingsboard.
set 22 21:13:36 kerberos thingsboard.jar[36912]:   ______    __      _                              ____                               __
set 22 21:13:36 kerberos thingsboard.jar[36912]:  /_  __/   / /_    (_)   ____    ____ _   _____   / __ )  ____   ____ _   _____  ____/ /
set 22 21:13:36 kerberos thingsboard.jar[36912]:   / /     / __ \  / /   / __ \  / __ `/  / ___/  / __  | / __ \ / __ `/  / ___/ / __  /
set 22 21:13:36 kerberos thingsboard.jar[36912]:  / /     / / / / / /   / / / / / /_/ /  (__  )  / /_/ / / /_/ // /_/ /  / /    / /_/ /
set 22 21:13:36 kerberos thingsboard.jar[36912]: /_/     /_/ /_/ /_/   /_/ /_/  \__, /  /____/  /_____/  \____/ \__,_/  /_/     \__,_/
set 22 21:13:36 kerberos thingsboard.jar[36912]:                               /____/
set 22 21:13:36 kerberos thingsboard.jar[36912]:  ===================================================
set 22 21:13:36 kerberos thingsboard.jar[36912]:  :: ThingsBoard ::       (v3.3.1)
set 22 21:13:36 kerberos thingsboard.jar[36912]:  ===================================================
lines 1-20/20 (END)

gedit /var/log/thingsboard/thingsboard.log
2021-09-22 21:18:46,136 [background-preinit] INFO  o.h.validator.internal.util.Version - HV000001: Hibernate Validator 6.0.13.Final
2021-09-22 21:18:46,223 [main] INFO  o.t.s.ThingsboardServerApplication - Starting ThingsboardServerApplication v3.3.1 on kerberos with PID 37255 (/usr/share/thingsboard/bin/thingsboard.jar started by thingsboard in /usr/share/thingsboard/bin)
2021-09-22 21:18:46,224 [main] INFO  o.t.s.ThingsboardServerApplication - No active profile set, falling back to default profiles: default
2021-09-22 21:18:47,173 [background-preinit] WARN  o.s.h.c.j.Jackson2ObjectMapperBuilder - For Jackson Kotlin classes support please add "com.fasterxml.jackson.module:jackson-module-kotlin" to the classpath

**Comment by Backdraft007** [comment]
Well, I think it is running.
Waht sasy `sudo netstat -tulpn`?

**Comment by gilbertoe** [comment]
Hi
The result of sudo netstat -tulpn is below...

gilberto@kerberos:~$  sudo netstat -tulpn
Conexões Internet Ativas (somente servidores)
Proto Recv-Q Send-Q Endereço Local          Endereço Remoto         Estado       PID/Program name    
tcp        0      0 127.0.0.1:30800         0.0.0.0:*               OUÇA       1731/core           
tcp        0      0 127.0.0.1:30900         0.0.0.0:*               OUÇA       1731/core           
tcp        0      0 127.0.0.53:53           0.0.0.0:*               OUÇA       840/systemd-resolve 
tcp        0      0 127.0.0.1:631           0.0.0.0:*               OUÇA       65223/cupsd         
tcp        0      0 127.0.0.1:5432          0.0.0.0:*               OUÇA       1147/postgres       
tcp        0      0 0.0.0.0:1883            0.0.0.0:*               OUÇA       1187/mosquitto      
tcp6       0      0 ::1:631                 :::*                    OUÇA       65223/cupsd         
tcp6       0      0 :::1883                 :::*                    OUÇA       1187/mosquitto      
udp        0      0 127.0.0.53:53           0.0.0.0:*                           840/systemd-resolve 
udp        0      0 0.0.0.0:631             0.0.0.0:*                           65226/cups-browsed  
udp        0      0 0.0.0.0:50226           0.0.0.0:*                           849/avahi-daemon: r 
udp        0      0 224.0.0.251:5353        0.0.0.0:*                           3376/chrome --enabl 
udp        0      0 0.0.0.0:5353            0.0.0.0:*                           849/avahi-daemon: r 
udp6       0      0 :::48935                :::*                                849/avahi-daemon: r 
udp6       0      0 fe80::65b0:9ef1:cca:546 :::*                                856/NetworkManager  
udp6       0      0 :::5353                 :::*                                849/avahi-daemon: r

**Comment by ashvayka** [comment]
ThingsBoard is probably not starting because you have mosquitto that occupied port 1883. You may reconfigure ThingsBoard or stop mosquitto.

**Comment by gilbertoe** [comment]
Hi
I removed mosquitto, but the problem remains.
gilberto@kerberos:~$ sudo netstat -tulpn
Conexões Internet Ativas (somente servidores)
Proto Recv-Q Send-Q Endereço Local          Endereço Remoto         Estado       PID/Program name    
tcp        0      0 127.0.0.1:30800         0.0.0.0:*               OUÇA       1731/core           
tcp        0      0 127.0.0.1:30900         0.0.0.0:*               OUÇA       1731/core           
tcp        0      0 127.0.0.53:53           0.0.0.0:*               OUÇA       840/systemd-resolve 
tcp        0      0 127.0.0.1:631           0.0.0.0:*               OUÇA       65223/cupsd         
tcp        0      0 127.0.0.1:5432          0.0.0.0:*               OUÇA       1147/postgres       
tcp6       0      0 ::1:631                 :::*                    OUÇA       65223/cupsd         
udp        0      0 127.0.0.53:53           0.0.0.0:*                           840/systemd-resolve 
udp        0      0 0.0.0.0:631             0.0.0.0:*                           65226/cups-browsed  
udp        0      0 0.0.0.0:50226           0.0.0.0:*                           849/avahi-daemon: r 
udp        0      0 224.0.0.251:5353        0.0.0.0:*                           3376/chrome --enabl 
udp        0      0 0.0.0.0:5353            0.0.0.0:*                           849/avahi-daemon: r 
udp6       0      0 :::48935                :::*                                849/avahi-daemon: r 
udp6       0      0 fe80::65b0:9ef1:cca:546 :::*                                856/NetworkManager  
udp6       0      0 :::5353                 :::*                                849/avahi-daemon: r 
The connection @ http://localhost:8080 is always refused by the browser. I tried with both Firefox and Chrome.
